src/user-script-obj.js

```diff
@@ -173,9 +173,10 @@ window.RemoteUserScript = class RemoteUserScript {
   }
 
   toString() {
-    return '[Greasemonkey Script ' + this.id
-        + (this.version ? '; ' + this.version : '')
-        + ']';
+    const fullScriptId = (this.version
```

Comment:
1) This should be formatted like

    const fullScriptId =
        this.version ? _('$1; Version $2', this.id, this.version) : this.id; 

2) This is not an "id".  We already call very different things "id".
3) I might actually prefer:

    if (this.version) {
      return _('[Greasemonkey Script $1; version $2]', this.id, this.version);
    } else {
      return _('[Greasemonkey Script $1]', this.id);
    }

The localized strings are more self-evident.  And IMO more readable.

Reply:
I like the third version best.